maro/cli/inspector/env_data_process.py

```diff
@@ -0,0 +1,201 @@
+        --index_name_mapping file: record the relationship between an index and its name.
+        type of this file varied between scenario.
+
+        summary file would be generated after data processing.
+
+    Args:
+        source(str): Data folder path.
+        force(str): expected input is yes/no. Indicates whether regenerate data.
+        **kwargs:
+
+    """
+    source_path = source
+    FORCE = force
+    if not os.path.exists(os.path.join(source_path, "manifest.yml")):
+        raise CliException("Manifest file missed. ")
+        os._exit(0)
+    manifest_file = open(os.path.join(source_path, "manifest.yml"), "r")
+    props_origin = manifest_file.read()
+    props = yaml.load(props_origin, Loader=yaml.FullLoader)
+    scenario = GlobalScenarios[str(props["scenario"]).upper()]
+    conversion_path = str(props["mappings"])
+    epoch_num = int(props["dump_details"]["epoch_num"])
+    prefix = props["dump_details"]["prefix"]
+
+    if not os.path.exists(source_path):
+        raise CliException("input path is not correct. ")
+    elif not os.path.exists(os.path.join(source_path, prefix + "0")):
+        raise CliException("No data under input folder path. ")
+
+    if FORCE == "true":
+        logger.info("Dashboard Data Processing")
+        _get_holder_name_conversion(scenario, source_path, conversion_path)
+        logger.info_green("[1/2]:Generate Name Conversion File Done.")
+        logger.info_green("[2/2]:Generate Summary.")
+        _generate_summary(scenario, source_path, prefix)
+        logger.info_green("[2/2]:Generate Summary Done.")
+
+    elif FORCE == "false":
+        logger.info_green("Skip Data Generation")
+        if not os.path.exists(os.path.join(source_path, Gfiles.name_convert)):
+            raise CliException("Have to regenerate data. Name Conversion File is missed. ")
+
+        if scenario == GlobalScenarios.CIM:
+            if not os.path.exists(os.path.join(source_path, Gfiles.ports_sum)):
+                raise CliException("Have to regenerate data. Summary File is missed. ")
+        elif scenario == GlobalScenarios.CITI_BIKE:
+            if not os.path.exists(os.path.join(source_path, Gfiles.stations_sum)):
+                raise CliException("Have to regenerate data. Summary File is missed. ")
+
+    launch_dashboard(source_path, scenario, epoch_num, prefix)
+
+
+def _init_csv(file_path: str, header: list):
+    """Clean & init summary csv file.
+
+    Args:
+        file_path (str): summary file path.
+        header (list): expected header of summary file.
+    """
+    if os.path.exists(file_path):
+        os.remove(file_path)
+    with open(file_path, "w") as csvfile:
+        writer = csv.DictWriter(csvfile, fieldnames=header)
+        writer.writeheader()
+
+
+def _summary_append(
+        scenario: enumerate, dir_epoch: str,
+        file_name: str, header: list,
+        sum_dataframe: pd.DataFrame, i: int, output_path: str):
```

Comment:
index not i

BTW, what's the diffrence between ``epoch_index`` and this ``index``?